bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -113,6 +118,12 @@ def __execute_integtest_sh(self, endpoint, port, security):
                 else self.repo.dir
             )
             (status, stdout, stderr) = execute(cmd, work_dir, True, False)
+            results_dir = os.path.join(
+                work_dir, "build", "reports", "tests", "integTest"
+            )
+            test_recorder_builder = TestRecorderBuilder(self.component.name, test_config, status, stdout, stderr,
+                                                        walk(results_dir), "S3")
```

Comment:
Hardcoded to 's3', could this be part of a separate configuration that can determine where recorded results code?

Reply:
[Discussed Offline] We should just remove, if we need to message anything 'special' just add a logfile "The data isn't here, its in a S3 bucket or cloudwatch log group {{ARN}}"